### ISSUE TITLE
使用paddlelite转.nb文件失败

### ISSUE THREAD

**Issue by PQ-girl** [PaddlePaddle/Paddle#43280]
### 问题描述 Please describe your issue

您好！我使用paddlelite将paddle模型转换为.nb文件时报错Error: This model is not supported, because 1 ops are not supported on 'arm'. These unsupported ops are: 'pad'。然后去paddlelite下提问后说：arm 上目前只支持pad2d 和 pad3d OP，可以将pad OP 转换为pad2d/pad3d 进行推理。
所以我想问一下如何将pad OP转换为pad2d 和 pad3d OP，麻烦你们帮帮我

**Comment by SunNy820828449** [comment]
已经反馈给相关同学，会尽快为你处理。

**Comment by SunNy820828449** [comment]
Pad1D的padding参数指定1D时的左右。Pad2D的padding参数指定2D时的左右、上下。假设用户原先Pad1D进行padding的参数为[pad_left, pad_rigth] 只需把Pad1D替换为Pad2D，其的padding参数改为[pad_left, pad_rigth, 0, 0] 即可。

参考文档：
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Pad1D_cn.html#pad1d
https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Pad2D_cn.html#pad2d

**Comment by SunNy820828449** [comment]
建议在训练阶段将pad替换为pad2d。

**Comment by PQ-girl** [comment]
我这个模型是用pytorch训练好的，然后通过onnx转为paddle模型，最后导出.nb文件时报错。我去pytorch训练脚本里看，发现这句代码transforms.Pad(4),有pad，它里面只传入了一个参数，我目前已经用pytorch训练好了所有模型，请问我还有其他办法改吗？还有要是在训练脚本里改是将transforms.Pad(4)直接改为transforms.Pad2d(4)就可以吗？
还有模型的定义里的conv2d = nn.Conv2d(in_channels, v, kernel_size=3, padding=1, bias=False)这个padding算吗
